src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -20,22 +22,22 @@
     public static Person[] getSamplePersons() {
         return new Person[] {
             new Person(new Name("Alex Yeoh"), new Phone("87438807"), new Email("alexyeoh@example.com"),
-                new Address("Blk 30 Geylang Street 29, #06-40"),
+                new Address("Blk 30 Geylang Street 29, #06-40"), new Schedule(new ArrayList<>()),
```

Comment:
Since Schedule is empty and shared with multiple people, we can have a constant variable named `EMPTY_SCHEDULE` and then use that instead.